src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.LeakDetection/PoisonType.cs

```diff
@@ -11,5 +11,6 @@ internal enum PoisonType
         Hash = 1,
         AssemblyAttribute = 2,
         NupkgFile = 4,
+        ReferenceAssemblyAttribute = 8,
```

Comment:
This feels too abstract.  There are types of reference assemblies that are allowed.  This poison type is specific to SBRP.  Because of that, I tend to think naming this SourceBuildReferenceAssembly would be better. (Note: I don't think the attribute suffix is necessary)

Reply:
Updated in https://github.com/dotnet/installer/pull/17339/commits/43f405c0aab49c27128276c44013c197b8fe9ab4